src/ui/components/confirm_transaction.h

```diff
@@ -43,4 +43,13 @@ component_t* confirm_transaction_fee_create(
     void (*confirm_callback)(void),
     void (*cancel_callback)(void));
 
+/**
+ * Creates a confirm screen.
+ */
+component_t* confirm_transaction_locktime_create(
```

Comment:
move to a new file, `confirm_locktime.h`.

Reply:
Should I really move this? It displays only in combination with the other two functions in there and I'll have to redefine the functions internal to that file.

Reply:
Hang on why don't you just use the already existing `workflow_confirm()`?

Reply:
Using it now.